package.json

```diff
@@ -13,7 +13,8 @@
     "nsp": "nsp check",
     "build": "npm run standard && npm run retire && npm run nsp",
     "postinstall": "bower install",
-    "benchmark": "node benchmark/homepage.js"
+    "benchmark": "node benchmark/homepage.js",
+    "seed": "node  ./src/config/mongoose_seed/configuration"
```

Comment:
Hi @kevinpr007,

I checked out this branch and noticed that NODE_ENV needs to set before running this command. Make sure to set variable inside script.

P.D.
It will be a temporary change until we merge the TDD brach which contains the cross-env package.